dtls.h

```diff
@@ -324,8 +324,16 @@ void dtls_check_retransmit(dtls_context_t *context, clock_time_t *next);
 #define DTLS_CT_HANDSHAKE          22
 #define DTLS_CT_APPLICATION_DATA   23
 
+#ifdef __GNUC__
+#define PACK( __Declaration__ ) __Declaration__ __attribute__((__packed__))
+#endif
+
+#ifdef _MSC_VER
+#define PACK( __Declaration__ ) __pragma( pack(push, 1) ) __Declaration__ __pragma( pack(pop))
+#endif
+
```

Comment:
Before this, the default, (may be a failing one) was `__attribute__((__packed__))`.
I would go for an explaining "#error", if PACK is not defined.

Reply:
Should be addressed in 7126ffca98564345c05bb5cf6fb69e4b67784ac4, maybe the error message could be improved, though.